### DISCUSSION TITLE
which version should be used for Cyberpunk 1.63?

### DISCUSSION THREAD

**Discussion by UzvarinatoR** [jac3km4/redscript#101] [Q&A]
Ok so i haven't updated to 2.0 deliberately, for many reasons, so i am staying on old 1.63 version.
But i tried many old versions and still can't find which one has been working in times of Cyberpunk 1.63?
The redmod versions i tried:
v0.5.16
v0.5.15
v0.5.14
v0.5.13
v0.5.12
v0.5.11

And all of them gives me Compilation Error message when i start the game.

**Comment by LazerDickMcCheese** [comment]
@UzvarinatoR, I know this is years old, but it's still relevant to people like me who refuse the v2.x _overhauls_! redscript silently assumes you keep the game update (as you and I have noticed), and v0.5.15 of rs is the first stable release to mention CP2077 v2.0 support. So, I'm assuming **v0.5.14 is the last Legacy-supported release**.

I've been troubleshooting my own mod list for a few hours now, and I believe this is the culprit. If anyone else comes to the same conclusion, please chime in or react.
